compliance/specs/exceptions/rfc5746/4.6.toml

```diff
@@ -0,0 +1,23 @@
+target = "https://tools.ietf.org/rfc/rfc5746#3.6"
+
+[[exception]]
```

Comment:
We have several other exceptions in our codebase. Is this how we're supposed to format them now?

Reply:
Sort of. This is how you write an exception that doesn't apply to any code. Like, these aren't requirements we've chosen to do the opposite of and can put that comment on the code where we do the opposite, these are requirements we're ignoring because they're not really relevant.